packages/js/src/executors/tsc/tsc.impl.ts

```diff
@@ -61,8 +69,8 @@ export function normalizeOptions(
 }
 
 export function createTypeScriptCompilationOptions(
-  normalizedOptions: NormalizedExecutorOptions,
-  context: ExecutorContext
+  context: ExecutorContext,
+  normalizedOptions: NormalizedExecutorOptions
```

Comment:
Since this function is exported, can we keep the options in the same order? If anyone is using this via a deep import this would break them and I don't see a benefit on our side from swapping it.